PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetProperties.h

```diff
@@ -0,0 +1,233 @@
+#ifndef ALIANALYSISTASKEMCALJETPROPERTIES_H
+#define ALIANALYSISTASKEMCALJETPROPERTIES_H
+/**
+ * \file AliAnalysisTaskEmcalJetProperties.h
+ * \brief Declaration of class AliAnalysisTaskEmcalJetProperties
+ *
+ * In this header file the class AliAnalysisTaskEmcalJetProperties is declared.
+ * This is a sample task that shows how to write a simple user analysis task
+ * using the EMCal jet framework. It is also used to do automatic benchmark
+ * tests of the software.
+ *
+ * \author Salvatore Aiola <salvatore.aiola@cern.ch>, Yale University
+ * \date Apr 27, 2016
+ */
+
+/* Copyright(c) 1998-2016, ALICE Experiment at CERN, All rights reserved. *
+ * See cxx source for full Copyright notice                               */
+
+// This code is the modified version of AliAnalysisTaskJetProperties.h
+
+
+#include <TF1.h>                      
+class TString;                        
+class TF1;                            
+class TH1F;
+class TH2F;
+class TH3F;
+class THnSparse;
+class TProfile;
+class AliRhoParameter;                
+
+#include "AliAnalysisTaskEmcalJet.h"
+#include "THistManager.h"
+#include "AliAnalysisTaskEmcalJetTagger.h" 
+
+/**
+ * \class AliAnalysisTaskEmcalJetProperties
+ * \brief Implementation of a sample jet analysis task.
+ *
+ * This class in an implementation of a sample task for EMCal jet analysis.
+ * It derives from AliAnalysisTaskEmcalJet.
+ */
+
+class AliAnalysisTaskEmcalJetProperties : public AliAnalysisTaskEmcalJet {
+ public:
+  
+  AliAnalysisTaskEmcalJetProperties();
+  AliAnalysisTaskEmcalJetProperties(const char *name);
+  virtual ~AliAnalysisTaskEmcalJetProperties();
+  
+  void UserCreateOutputObjects();
+  void Terminate(Option_t *option);
+  
+  Double_t BkgSubtracted(Int_t KtJetCont, Int_t AktJetCont);
+  Double_t GetNch(Bool_t Isrec);
+  Double_t GetNchUEPC(Bool_t Isrec);
+  Double_t GetFF(Bool_t Isrec, AliEmcalJet *jet, Int_t iTrack);
+  Double_t GetFFUEPC(Bool_t Isrec, AliEmcalJet *jet, TList *UETrkList, Int_t iTrack);
+  Double_t* GetPtSum(Bool_t Isrec, AliEmcalJet *jet, Double_t* PtSum);
+  Double_t* GetPtSumUE(Bool_t Isrec, TList *UETrkList, Double_t etaTilted, Double_t phiTilted, Double_t* PtSum);
+  Int_t GetMLeadingJet();
+  
+  void SetJetContainerBase(Int_t c){ fContainerBase = c;}
+  void SetJetContainerTag(Int_t c){ fContainerTag  = c;}
+  Int_t fContainerBase = -1;
+  Int_t fContainerTag = -1;
+  
+  Int_t fContainerKtRec = -1;
+  Int_t fContainerKtGen = -1;
+  Int_t fContainerAktRec = -1;
+  Int_t fContainerAktGen = -1;
```

Comment:
These attributes should be protected or private

Reply:
These should be protected. I have changed accordingly.

Reply:
Thanks!